hack/build-manifests.sh

```diff
@@ -18,96 +18,190 @@ CONTAINER_PREFIX="${CONTAINER_PREFIX:-kubevirt}"
 CNA_CONTAINER_PREFIX="${CNA_CONTAINER_PREFIX:-quay.io/kubevirt}"
 IMAGE_PULL_POLICY="${IMAGE_PULL_POLICY:-IfNotPresent}"
 
-#CNV IMS Images
+# Component Images
+KUBEVIRT_IMAGE="${KUBEVIRT_IMAGE:-docker.io/kubevirt/virt-operator:v0.20.8}"
```

Comment:
Can we query the respective repos in git for this value?

Reply:
I think that's something we could pursue but I think we should create an issue/card for that as an improvement to make.

Reply:
I don't know if the docker/quay apis have a good way of getting the latest released tag...plus I think we would want to have reproducible builds (not ones that change based on when you run it like in the case KubeVirt does a new release).

Basically, I think it requires enough discussion and thought that I believe it's worth tackling separately from this PR.